hibernate-ehcache/src/main/java/org/hibernate/cache/ehcache/MissingCacheStrategy.java

```diff
@@ -45,7 +45,8 @@ public static MissingCacheStrategy interpretSetting(Object value) {
 
 		if ( StringHelper.isEmpty( externalRepresentation ) ) {
 			// Use the default
-			return MissingCacheStrategy.FAIL;
+			// Default is CREATE_WARN for backward compatibility reasons; we should switch to FAIL at some point.
+			return MissingCacheStrategy.CREATE_WARN;
```

Comment:
OK, I think even having warnings for a perfectly valid configuration is not acceptable.

We should probably discuss that with the others.

Reply:
It's a warning that you can disable forever with a configuration property, and we don't know that the configuration is valid, it depends on the JCache implementation.
Note that if you don't want warnings either by default, then the whole pull request and the changes we discussed become useless, since we could just have switched the behavior to "create" and not provided any configuration option.